src/snowflake/snowpark/column.py

```diff
@@ -605,7 +605,7 @@ def __str__(self):
         return f"Column[{self._expression}]"
 
     def __repr__(self):
-        return f"Column({self._expression})"
+        return f"Column({self._expression})"  # pragma: no cover
```

Comment:
The onlt test I could think of for `__repr__` is `assert eval(repr(x)) == x`, which does not apply for our case.

Reply:
Should we change to use `__str__`. It shouldn't be a breaking change